charts/ccd-docker-user-profile-importer/templates/_helpers.tpl

```diff
@@ -0,0 +1,14 @@
+{{- define "importer.definition.resourcegroup" }}
```

Comment:
Is this used?

Reply:
not required as i removed ref in ci-values